business_requirement_deliverable_project/models/project.py

```diff
@@ -26,15 +26,11 @@ def generate_project_wizard(self):
                     config settings"""))
         lines = []
         for br in br_ids:
-            if br.state not in [
-                    'stakeholder_approval',
-                    'cancel',
-                    'done'
-            ]:
+            if br.state not in ['stakeholder_approved', 'cancel', 'done']:
                 raise ValidationError(
-                    _("""All business requirements of the project should
-                        be Approved by Stakeholder"""))
-            if br.state != 'stakeholder_approval':
+                    _("All business requirements of the project should "
+                      "be stakeholder_approved/canceled/done"))
+            if br.state not in ['stakeholder_approved', 'cancel', 'done']:
                 continue
```

Comment:
@victormartinelicocorp I think this piece of code will never get executed as we already checked the status before and raised the exception. So I guess we should remove the conditions that is using continue statement.
Let me know your opinion.